x-pack/plugins/ml/server/models/data_recognizer/modules/security_auth/ml/auth_rare_source_ip_for_a_user.json

```diff
@@ -1,24 +1,18 @@
 {
   "description": "Security: Authentication - looks for a user logging in from an IP address that is unusual for the user. This can be due to credentialed access via a compromised account when the user and the threat actor are in different locations. An unusual source IP address for a username could also be due to lateral movement when a compromised account is used to pivot between hosts.",
```

Comment:
Probably worth changing the case of `looks` in this set of edits for consistency to `Looks`.

Reply:
Changed!